Cargo.lock

```diff
@@ -2975,6 +2975,14 @@ dependencies = [
  "zenoh_backend_traits",
 ]
 
+[[package]]
+name = "zenoh-plugin-trait"
+version = "0.1.0-dev"
```

Comment:
Version should be the same as zenoh (0.5.0-dev) since we will release all together.